src/main/scala/nl/knaw/dans/api/sword2/DepositProperties.scala

```diff
@@ -25,9 +26,9 @@ import scala.util.Try
 
 object DepositProperties {
   val log = LoggerFactory.getLogger(getClass)
-  case class State(label: String, description: String, timeStamp: String)
+  case class Statement(label: String, description: String, timeStamp: String)
```

Comment:
Okay, I can see the problem with the name clash, but this isn't a proper solution either. Please, try to solve the deeper problem, not just the name clash, or everything will just get more confusing.